cityofaustin/atd-vz-data
# Fill fatalities table with data from existing crash information

## Associated issues
Closes cityofaustin/atd-data-tech/issues/11486

## Testing
**URL to test:** 
Test this with a local VZ DB environment setup

**Steps to test:**
1. Checkout this branch locally
2. Run the queries in [fatalities_backfill.sql](https://github.com/cityofaustin/atd-vz-data/pull/1181/files) within your local VZ DB to populate the fatalities table with data
3. All persons and primary persons with a `prsn_injry_sev_id` of 4 should be present in the fatalities table.

Not sure if putting this script in the atd-toolbox folder long-term makes sense but just put it here for now for y'alls ease of testing.

---
#### Ship list
